src/main/kotlin/nmcp/NmcpPublishTask.kt

```diff
@@ -48,7 +52,7 @@ abstract class NmcpPublishTask : DefaultTask() {
         val body = MultipartBody.Builder()
             .addFormDataPart(
                 "bundle",
-                "publication.zip",
+                "${publicationName.orElse("publication")}.zip",
```

Comment:
Should we default to `${project.name}-${project.version}.zip` instead?

Reply:
Honestly that is even better yes. Just pushed it